### PR TITLE
Conwomap 5

### LINKED ISSUE #83: conwomap_5
Description: An example of a contour plot with a logarithmic Y axis.

Script URL: https://www.ncl.ucar.edu/Applications/Scripts/conwomap_5.ncl
Plot URL: https://www.ncl.ucar.edu/Applications/Images/conwomap_5_2_lg.png

Category: Contours without maps

### PR DESCRIPTION
**Pull request by hCraker**
Closes #83 
Plot so far:
image

### CODE REVIEW

**Comment by hCraker** [comment]
All of the formatting for the axes. titles, and color bar are finished, but the plot doesn't match what is in the NCL example. It seems like the NCL code is using a function called `vinth2p()` to do some sort of transformation on the data. 

I saw in geocat-comp that porting this into Python is an issue in progress. @clyne @erogluorhan I saw your names on that issue. Am I correct that it still in the works? If so, I think this pull request will have to be put on hold until an equivalent Python function is created.

**Comment by erogluorhan** [comment]
> All of the formatting for the axes. titles, and color bar are finished, but the plot doesn't match what is in the NCL example. It seems like the NCL code is using a function called `vinth2p()` to do some sort of transformation on the data.
> 
> I saw in geocat-comp that porting this into Python is an issue in progress. @clyne @erogluorhan I saw your names on that issue. Am I correct that it still in the works? If so, I think this pull request will have to be put on hold until an equivalent Python function is created.

Yes, your understanding is correct! **vinth2p** is still a work in progress, so we would better put this PR on hold as a draft for a while.

**Comment by erogluorhan** [comment]
@hCraker `vinth2p` function is now available via `geocat.comp` with its new name `interp_hybrid_to_pressure`. Please let me know if you would be willing to continue doing this PR by yourself or hand it over to me. Either works fine with me.

**Comment by hCraker** [comment]
> @hCraker `vinth2p` function is now available via `geocat.comp` with its new name `interp_hybrid_to_pressure`. Please let me know if you would be willing to continue doing this PR by yourself or hand it over to me. Either works fine with me.

@erogluorhan That's great news! I can finish up this PR.

**Comment by erogluorhan** [comment]
> > @hCraker `vinth2p` function is now available via `geocat.comp` with its new name `interp_hybrid_to_pressure`. Please let me know if you would be willing to continue doing this PR by yourself or hand it over to me. Either works fine with me.
> 
> @erogluorhan That's great news! I can finish up this PR.

Sounds good, thank you!

**Comment by hCraker** [comment]
@erogluorhan Do you know if there is an example script showing how to use `interp_hybrid_to_pressure`? I'm having issues where the function outputs values that are nonsensical when plotted.

**Comment by erogluorhan** [comment]
> @erogluorhan Do you know if there is an example script showing how to use `interp_hybrid_to_pressure`? I'm having issues where the function outputs values that are nonsensical when plotted.

Check out the commented out test case in the very end of [test_interp_hybrid_to_pressure](https://github.com/NCAR/geocat-comp/blob/main/test/test_interp_hybrid_to_pressure.py) of geocat-comp. It is indeed conwomap_5 code that works well with `interp_hybrid_to_pressure`, which might need some additional polishings of yours to have an identical plot to NCL though.

**Comment by hCraker** [comment]
@erogluorhan I'm not sure why the builds are failing, but the code itself is fine. I think the issue has to do with the GitHub actions, but I'm inexperienced with those. Could you or someone else lend me a hand with this? cc: @michaelavs @anissa111

**Comment by anissa111** [comment]
> @erogluorhan I'm not sure why the builds are failing, but the code itself is fine. I think the issue has to do with the GitHub actions, but I'm inexperienced with those. Could you or someone else lend me a hand with this? cc: @michaelavs @anissa111

You're right, it seems to just be the pre commit hooks. They're funny to get set up. How much of them do you have set up on local?

**Comment by anissa111** [comment]
That should make the build pass, but it messed up the imports. @michaelavs was talking about this with other examples, too. The isort hook is pulling some imports above the import comment line.

**Comment by hCraker** [comment]
> You're right, it seems to just be the pre commit hooks. They're funny to get set up. How much of them do you have set up on local?

To be completely honest, I'm not even sure what pre commit hooks are or if I have any of them set up locally. I've never heard of them before.

**Comment by anissa111** [comment]
> To be completely honest, I'm not even sure what pre commit hooks are or if I have any of them set up locally. I've never heard of them before.

That's fine! I imagine you'll have to learn them next week anyway, but for now, they're checks that run locally when you make a commit that do stuff like formatting automatically. We have the precommit hooks we want set up already, so you just need to install them for the repo locally. [This website](https://pre-commit.com/) is useful. 

In summary, you can install precommits with `pip install pre-commit`. Then, from the root of the GeoCAT-examples repository, you're going to want to run `pre-commit install`. This will get the precommits we defined set up on your local. Then, when you make a commit, a bunch of checks will run. If the checks fail, you'll need to re-add the altered file to the commit and try again. You can also trigger the hooks manually with `pre-commit run --all-files`

**Comment by hCraker** [comment]
@anissa111 I got the build issues fixed. Thanks for your help. This PR is ready to be reviewed when you have some time.

File: Plots/Contours/NCL_conwomap_5.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
+"""
+NCL_conwomap_5.py
+=================
+This script illustrates the following concepts:
+   - Drawing a simple contour plot
+   - Making an axis logarithmic in a contour plot
+   - Changing the labels and tickmarks on a contour plot
+   - Creating a main title
+   - Using the geocat-comp method interp_hybrid_to_pressure
```
Comment (erogluorhan) [inline Plots/Contours/NCL_conwomap_5.py:9]: Would be great to provide an embedded link to documentation of this function here. The link should be:

https://geocat-comp.readthedocs.io/en/latest/user_api/generated/geocat.comp.interp_hybrid_to_pressure.html#geocat.comp.interp_hybrid_to_pressure

**Review by erogluorhan — APPROVED** [review]
This looks great! Thanks for having this done here despite your initial days in GeoCAT-comp :)

Could you please address a small inline request?

**Pull request merged**